documentation/decisions/0003-styling-rules.md

```diff
@@ -23,15 +23,17 @@ Tailwind gives us opportunity to deliver fast and well organize styles in the ap
 - Avoid using inline styles - if you have nonstandard value for some dimension you can pass it in the class (`px-[17px]`).
 - If you have conditional styling and need to group you styles you can use custom classes which applies tailwind classes:
 
-```
+```css
 .custom-class {
   @apply flex px-8 bg-red;
 }
 ```
 
 - Creating new common component first check if HeadlessUI offers similar solution and use it if possible.
-- If you want to handle `isLoading` state in some component you can use `v-loading` directive - it will do the job for you:
+- If you want to visually handle `isLoading` state in some component you can use `v-loading` directive - it will do the job for you:
 
+```vue
+<div v-loading="isLoading">My div</div>
 ```
-<div v-loading="isLoading" >My div</div>
-```
+
+It add class `v-loading` with loading spinner animation. With most cases it should work without any additional styling. If you meet some trouble with it check you custom pseudoelements (`:before`, `:after`) or `min-height` of your element. They might override styles from `v-loading`.
```

Comment:
sure, I will add `will` 😄 

You are right, `absolute` elements will align to parents with such positions as you wrote. I think we don't need to care on it on this moment. In most cases if we would like to add loader to some element it would be positioned correctly 👌

Reply:
Thank you! 😄